README.rst

```diff
@@ -2,6 +2,15 @@
 pyface: traits-capable windowing framework
 ==========================================
 
+.. image:: https://travis-ci.org/enthought/pyface.svg?branch=master
+    :target: https://travis-ci.org/enthought/pyface
+
+.. image:: https://ci.appveyor.com/api/projects/status/7xxghu5phbvml75j/branch/master?svg=true
+    :target: https://ci.appveyor.com/project/itziakos/pyface/history
```

Comment:
Is it possible to have this point at an 'enthought' URL, rather than one attached to your account?